src/ClientGenerator/ClientGenerator.csproj

```diff
@@ -14,7 +14,7 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>Orleans.CodeGeneration</RootNamespace>
     <AssemblyName>ClientGenerator</AssemblyName>
-    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.5.1</TargetFrameworkVersion>
     <FileAlignment>512</FileAlignment>
     <PublishUrl>publish\</PublishUrl>
     <Install>true</Install>
```

Comment:
Sorry for being "negative" here, but everyone taking apart a csproj, and it disturbs noone that clickonce related stuff from .net 2.0 present is in the csproj files. Just to put some light on the other side of the coin. ;-(